client/plots/singleCellPlot.js

```diff
@@ -118,13 +120,15 @@ class singleCellPlot {
 			label: 'Gene Expression',
 			id: GENE_EXPRESSION_TAB,
 			active: activeTab == GENE_EXPRESSION_TAB,
+			isVisible: () => initialSample || this.state?.config.sample,
```

Comment:
I think this should be `state.config.sample !== undefined`, and not have to use an `initialSample` variable. Similar for the other `isVisible()` callbacks below.